lib/stat_tracker.rb

```diff
@@ -0,0 +1,253 @@
+require 'csv'
+require_relative 'game.rb'
+require_relative 'team.rb'
+require_relative 'game_stat.rb'
+require 'pry'
+
+class StatTracker
+  attr_reader :games, :teams, :game_stats 
+
+  def initialize(games, teams, game_stats)
+    @games = games
+    @teams = teams
+    @game_stats = game_stats
+  end
+
+  def self.from_csv(locations)
+
+    games_data = CSV.read(locations[:games], { encoding: "UTF-8", headers: true, header_converters: :symbol, converters: :all})
+    teams_data = CSV.read(locations[:teams], { encoding: "UTF-8", headers: true, header_converters: :symbol, converters: :all})
+    game_teams_data = CSV.read(locations[:game_teams], { encoding: "UTF-8", headers: true, header_converters: :symbol, converters: :all})
```

Comment:
Extract a constant for `CSV_CONFIG`

And I think that we have an option to lift it directly into a hash.